docs/source/reference/api-only.md

```diff
@@ -68,36 +55,29 @@ The default is `/` which means that the Hub will receive all requests not alread
 
 There are three values that are most logical for `hub_routespec`:
 
-- `/` - this is the default, and used in most deployments.
-  It is also the only option prior to JupyterHub 1.4.
-- `/hub/` - this serves only Hub pages, both UI and API
-- `/hub/api` - this serves _only the Hub API_, so all Hub UI is disabled,
-  aside from the OAuth confirmation page, if used.
+- `/` - This is the default, and used in most deployments. It is also the only option prior to JupyterHub 1.4.
+- `/hub/` - This serves only Hub pages, both UI and API
+- `/hub/api` - This serves _only the Hub API_, so all Hub UI is disabled. Aside from the OAuth confirmation page, if used.
 
-If you choose a hub routespec other than `/`,
-the main JupyterHub feature you will lose is the automatic handling of requests for `/user/:username`
-when the requested server is not running.
+If you choose a hub routespec other than `/`, the main JupyterHub feature you will lose is the automatic handling of requests for `/user/:username` when the requested server is not running.
 
-JupyterHub's handling of this request shows this page,
-telling you that the server is not running,
-with a button to launch it again:
+JupyterHub's handling of this request shows this page, telling you that the server is not running, with a button to launch it again:
```

Comment:
Most of the changes here are to re-wrap text, which isn't an improvement. The lines are wrapped [on clauses or sentences](https://sembr.org).

We don't enforce that because I don't like to enforce any style that can't be autoformatted, but I'd prefer to avoid pure style changes that _remove_ the preferred style.